packages/react-docsite-components/.eslintrc.json

```diff
@@ -2,6 +2,7 @@
   "extends": ["plugin:@fluentui/eslint-plugin/react--legacy"],
   "root": true,
   "rules": {
+    "import/no-webpack-loader-syntax": "off",
```

Comment:
Why did you turn this off? Webpack assumptions actually create problems for some consumers of this package.

Reply:
For some reason, this started complaining everywhere after an update. I wasn't sure why and thought i had brought in something which changed the default.

Reply:
I just tried reverting it and running eslint, and it worked for me.

Reply:
i will remove this, thanks!

Reply:
you got it :D ty